main.py

```diff
@@ -101,6 +104,33 @@ def update_kontaktdaten_interactive(
             input_kontaktdaten_key(
                 kontaktdaten, ["kontakt", "notificationReceiver"], "> Mail: ")
 
+        if "notifications" not in kontaktdaten:
+            kontaktdaten["notifications"] = {}
+        if configure_notifications:
+            if "pushover_app_token" not in kontaktdaten["notifications"]:
+                if input("> Benachtigung mit Pushover einrichten? (y/n): ").lower() != "n":
+                    print()
+                    input_kontaktdaten_key(
+                        kontaktdaten, ["notifications", "pushover_app_token"],
+                        "> Geben Sie den Pushover APP Token ein: ")
+                    input_kontaktdaten_key(
+                        kontaktdaten, ["notifications", "pushover_user_key"],
+                        "> Geben Sie den Pushover User Key ein: ")
+                else:
+                    kontaktdaten["notifications"]["pushover_app_token"] = ""
+
+            if "telegram_api_token" not in kontaktdaten["notifications"]:
+                if input("> Benachtigung mit Telegram einrichten? (y/n): ").lower() != "n":
+                    print()
+                    input_kontaktdaten_key(
+                        kontaktdaten, ["notifications", "telegram_api_token"],
+                        "> Geben Sie den Telegram API Token ein: ")
+                    input_kontaktdaten_key(
+                        kontaktdaten, ["notifications", "telegram_chat_id"],
+                        "> Geben Sie die Telegram Chat ID ein: ")
+                else:
+                    kontaktdaten["notifications"]["telegram_api_token"] = ""
```

Comment:
Warum nicht unterteilen in ein `dict` für Pushover und eins für Telegram? Dann kann man gleich das korrekte `dict` an die Funktionen `pushover_notification` und `telegram_notification` übergeben.

Etwa so (untested):

```suggestion
            if "pushover" not in kontaktdaten["notifications"]:
                kontaktdaten["notifications"]["pushover"] = {}
                if input("> Benachtigung mit Pushover einrichten? (y/n): ").lower() != "n":
                    print()
                    input_kontaktdaten_key(
                        kontaktdaten, ["notifications", "pushover", "app_token"],
                        "> Geben Sie den Pushover APP Token ein: ")
                    input_kontaktdaten_key(
                        kontaktdaten, ["notifications", "pushover", "user_key"],
                        "> Geben Sie den Pushover User Key ein: ")

            if "telegram_api_token" not in kontaktdaten["notifications"]:
                kontaktdaten["notifications"]["telegram"] = {}
                if input("> Benachtigung mit Telegram einrichten? (y/n): ").lower() != "n":
                    print()
                    input_kontaktdaten_key(
                        kontaktdaten, ["notifications", "telegram", "api_token"],
                        "> Geben Sie den Telegram API Token ein: ")
                    input_kontaktdaten_key(
                        kontaktdaten, ["notifications", "telegram", "chat_id"],
                        "> Geben Sie die Telegram Chat ID ein: ")
```

Reply:
Ist das auch notwendig wenn jetzt beide Funktionen über eine Funktion ausgelöst werden? Der Hauptfunktion muss dann ja sowieso alles übergeben werden.

Reply:
Es hat halt den Vorteil, dass es dann schön getrennt ist.

Reply:
Habe die daten für telegram und pushover jetzt in 2 dicts getrennt